docs/version0.15.md

```diff
@@ -94,6 +95,12 @@ the JVM Tool Interface (JVMTI).
 
 The Java memory map (jmap) tool is similar to the HotSpot tool of the same name, and can be used to print statistics about classes on the heap, including the number of objects and their aggregate size. For usage information, see [Java memory map (jmap) tool](tool_jmap.md).
 
+### Automatically setting an initial heap size
+
+OpenJ9 can now learn and set an appropriate initial heap size for an application as an alternative to a user manually sizing and setting an `-Xms` value. The VM records the size of the heap when startup processing ends, writing this data to the shared classes cache. An average value is set over ~5 restarts, helping to ensure that the value used for the initial heap size is as accurate as possible. The heap size recorded is specific to the application command line, therefore a different hint is stored for every unique command line.
+
```

Comment:
like at the other spot down below, I'd rather avoid mentioning 5

Reply:
OK